src/masks.py

```diff
@@ -1,33 +1,45 @@
-def get_mask_card_number(card_number: int) -> str:
+def get_mask_card_number(card_number: int = None) -> str:
```

Comment:
Функция должна принимать номер карты в виде строки, причем он не может быть пустым
```suggestion
def get_mask_card_number(card_number: str) -> str:
```